src/world_interface/kinematic_common_interface.cpp

```diff
@@ -36,6 +38,14 @@ double setCmdVel(double dtheta, double dx) {
 		return 0;
 	}
 
+	control::swerve_rots_t curr_wheel_rots = {
+		robot::getMotorPos(motorid_t::frontLeftWheel).getData(),
+		robot::getMotorPos(motorid_t::frontRightWheel).getData(),
+		robot::getMotorPos(motorid_t::rearLeftWheel).getData(),
+		robot::getMotorPos(motorid_t::rearRightWheel).getData()};
```

Comment:
We should handle the case where any of these datapoints are invalid, otherwise it will crash if we haven't gotten an encoder packet yet. I added a PR (#324) that should make things easier so we can just add a try-catch around the whole block.

Reply:
Note that this should be done everywhere else we do a `getData()` without checking if the datapoint is valid.